pyfr/solvers/base/elements.py

```diff
@@ -173,8 +173,10 @@ def set_backend(self, backend, nscal_upts, nonce):
         if 'vect_fpts' in sbufs:
             self._vect_fpts = valloc('vect_fpts', nfpts)
 
+        self._shape = self._scal_upts.shape
```

Comment:
?

Reply:
Making shape and eles instance variables to make them easily available for the pseudointegrator:
   
       for ele in self.system.eles:
            # Allocate storage for previous error
            ele.err_prev = self.backend.matrix(ele._shape, np.ones(ele._shape),
                                               tags={'align'})

            # Append the error kernels to the proxylist
            self.pintgkernels['eles', 'localerrest'].append(errestkern(ele))